cmd/tidb-lightning-ctl/main.go

```diff
@@ -190,8 +190,55 @@ func checkpointRemove(ctx context.Context, cfg *config.Config, tableName string)
 		return errors.Trace(err)
 	}
 	defer cpdb.Close()
+	if err = cpdb.RemoveCheckpoint(ctx, tableName); err != nil {
```

Comment:
If the lightning-ctl exit before `cleanupMetas` but after `RemoveCheckpoint`, it can not retry because it will found the task id equals zero.